doc/source/whatsnew/v0.23.0.txt

```diff
@@ -966,6 +966,7 @@ Other API Changes
 - Constructing a Series from a list of length 1 no longer broadcasts this list when a longer index is specified (:issue:`19714`, :issue:`20391`).
 - :func:`DataFrame.to_dict` with ``orient='index'`` no longer casts int columns to float for a DataFrame with only int and float columns (:issue:`18580`)
 - A user-defined-function that is passed to :func:`Series.rolling().aggregate() <pandas.core.window.Rolling.aggregate>`, :func:`DataFrame.rolling().aggregate() <pandas.core.window.Rolling.aggregate>`, or its expanding cousins, will now *always* be passed a ``Series``, rather than a ``np.array``; ``.apply()`` only has the ``raw`` keyword, see :ref:`here <whatsnew_0230.enhancements.window_raw>`. This is consistent with the signatures of ``.aggregate()`` across pandas (:issue:`20584`)
+- Created the named argument `usecols_excel` that receives a string containing comma separated Excel ranges and columns or a list containing strings indicating Excel ranges and columns. The `usecols` named argument at :func:`read_excel` now receives a list of strings containing column labels or a list of integers representing column indexes or a callable or an int indicating the last column to be parsed. (:issue:`18273`)
```

Comment:
use double-backticks around the args (``usecols_excel``). 

start like this:

`` :func:`read_excel` has gained the keyword argument ``usecols_excel`` ``